azure_media_services/public/css/player.css

```diff
@@ -30,4 +30,19 @@
 .xmodule_display.xmodule_SequenceModule .sequence-nav ol li button.seq_video .icon:before,
 .xmodule_display.xmodule_SequenceModule .sequence-nav ol li button.seq_other .icon:before {
     content: "\f008" !important;
-}
\ No newline at end of file
+}
+
+/*google chrome compatibility*/
+.amp-default-skin.vjs-fullscreen {
+    height: 100% !important;
+}
+
+.xmodule_display.xmodule_VideoModule .video .video-wrapper {
+   margin-right: 0 !important;
+}
+
+/*closed caption overlay with control bar*/
+.azuremediaplayer .vjs-text-track-display {
+   bottom: 5em !important;
```

Comment:
5em feels like way too many. can we go with something like 1em or 2em instead? also, I recall that the demo website has the same issue. we may want to raise this concern with the azure media player team as well (especially if it isn't resolved in their next version).